src/v2/view-builder/views/AdminConsent.js

```diff
@@ -0,0 +1,82 @@
+import { loc, View, Model } from 'okta';
+import BaseView from '../internals/BaseView';
+import BaseForm from '../internals/BaseForm';
+import ScopeList from '../../../views/admin-consent/ScopeList';
+import consentLogoHeaderTemplate from '../../../views/shared/templates/consentLogoHeaderTemplate';
```

Comment:
Interesting. But we don't want to share code with v1. Right ? this resides outside of V2 folder. Mostly For such scenarios we just create another view. This template won't be reusable apart from consent views.

Reply:
Why not share view code? They're using the same UI framework. But I can also copy and move them to the v2 folder.

Reply:
Even with slightest difference most dev prefer to create their own templates, so it's generally not reusable most of the time.. Instead just create a plain view that returns you this screen in V2.